weave/panels_py/instructions/panel_trace_monitor.md

```diff
@@ -65,3 +74,8 @@ st.log(TraceSpanDict(
 ))
 
 ```
+
+## Analyze Data
+Once you've logged your data, use the browser on your left to find your table. From there, choose a template to get started!
+
+![](/docs/assets/traces_debug_board.png)
```

Comment:
this can't be relative, it needs to be absolute. please make ti reference github

Reply:
example: https://raw.githubusercontent.com/wandb/weave/master/docs/assets/traces_debug_board.png

Reply:
Resolved!